.ipynb_checkpoints/gensim Quick Start-checkpoint.ipynb

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
Thanks!
Removed it and committed!
Thanks for you kindness!
will move on to more difficult a task.
Speaking of which, could you please review the issue : 
https://github.com/RaRe-Technologies/gensim/issues/1419

It is about a word embedding visualiser that I have implemented.